### PR TITLE
[#50] Fixing fullPage

### PR DESCRIPTION
**Pull request by johnyvelho**
PR related to #50

### CODE REVIEW

**Comment by sindresorhus** [comment]
There's already a PR for this: https://github.com/sindresorhus/capture-website/pull/49 Would be great if you could help review it.

File: index.js [hunk 1]
```diff
@@ -347,8 +347,12 @@ const captureWebsite = async (input, options) => {
 			/* eslint-enable no-undef */
 		});
 
-		// Some extra delay to let images load
-		await page.waitForFunction(imagesHaveLoaded, {timeout: 60});
+		try {
+			// Some extra delay to let images load
+			await page.waitForFunction(imagesHaveLoaded, {timeout: timeoutInSeconds});
```
Comment (netrules) [inline index.js:352]: This function has been deprecated. Please check out the latest changes in https://github.com/sindresorhus/capture-website/pull/49

**Comment by johnyvelho** [comment]
@sindresorhus, @netrules  yeah, I checked before start it. honestly, I intended just to fix the zombie process. But since I got these other errors, I fixed. I believe what you are trying to do is a different feature. The tests are described as "lazy image loading" and that is what is supposed to do: scroll and let the images lazy images load/try in the timeout limit. When you set the capture-website settings, you can set a timeout, the client is expecting that this timeout is going to be accomplished. the current way is dealing with that, your changes could cause infinite processing. I would treat what you are trying to do as a different feature with automatic scroll, keep scrolling if news elements are attached to the documents, option to wait to do load all the images + xhr requests. but as I said, I truly believe this should be part of a new option/feature

**Comment by netrules** [comment]
@johnyvelho Could you fork that repository and see if the error persists?

**Comment by sindresorhus** [comment]
Closing as this is not the correct way to fix it. Closing in favor of https://github.com/sindresorhus/capture-website/issues/58.

**Pull request closed without merging**